TRIAGING.md

```diff
@@ -0,0 +1,80 @@
+## The Triaging Process at `conda`
+
+"Triaging" refers to the process of assessing the priority of incoming issues.
+
+In order to explain how various `conda` issues are evaluated, the following document will provide information about our triaging process in the form of an FAQ.
+
+
+### Why triage?
+
+At the most basic "bird's eye view" level, triaged issues will fall into the category of three main priority levels:
+
+- Do now
+- Do sometime
+- Never do (_i.e._, close)
+
+At its core, triaging enables new issues to be sorted into these three categories, which enables them to be processed at a velocity similar to or exceeding the rate at which new issues are coming in. One of the benefits of actively triaging issues is to avoid burnout and make necessary work sustainable; this is done by eliminating a never-ending backlog that has not been reviewed by any maintainers.
+
+There will always be broad-scope design and architecture implementations that the `conda` team will be interested in pursuing; by actively triaging issues, the `conda` engineers will be able to more easily track and tackle both specific and big-picture goals.
+
+
+### How do items show up in triaging?
+
+New issues that are opened in any of the repositories in the [`conda` GitHub project](https://github.com/conda) will show up in the `New` column of the [Triaging board](https://github.com/orgs/conda/projects/4).
```

Comment:
Let's briefly discuss how we are using GHA to orchestrate our project boards. Include links to the GH actions ([boards](https://github.com/conda/infra/blob/main/.github/workflows/boards.yml) & [issues](https://github.com/conda/infra/blob/main/.github/workflows/issues.yml)) for those who want the nitty-gritty details. Also mention how we view GHA defined in conda/infra to be canonical and we use the [sync workflow](https://github.com/conda/infra/blob/main/.github/workflows/sync.yml) to sync any modifications out to the repos.